crates/ruff_python_ast/ast.toml

```diff
@@ -0,0 +1,174 @@
+# This file is used by generate.py to autogenerate our Python AST data model.
+#
+# We have defined a Rust struct for each syntax node in `src/nodes.rs`.  Many of
+# these nodes belong to groups.  For instance, there is a `Stmt` group
+# consisting of all of the syntax nodes that represent valid Python statements.
+#
+# There is a special group named `ungrouped` that contains syntax nodes that do
+# not belong to any group.
+#
+# Each group is defined by two sections below.  The `[GROUP]` section defines
+# options that control the auto-generation for that group.  The `[GROUP.nodes]`
+# section defines which syntax nodes belong to that group.  The name of each
+# entry in the nodes section must match the name of the corresponding Rust
+# struct.  The value of each entry defines options that control the
+# auto-generation for that syntax node.
+#
+# The following group options are available:
+#
+# add_suffix_to_is_methods: [true/false]
+#   Controls the name of the is_foo methods of the group's enums.  If false (the
+#   default), these methods will use the variant name in snake_case.  If true,
+#   then the group prefix will be moved to the end before snake_casing.  (That
+#   is, `StmtIf` will become `if_stmt`.)
+#
+# anynode_is_label: foo_bar
+#   Controls the name of the AnyNode::foo_bar, AnyNode::is_foo_bar, and
+#   AnyNodeRef::is_foo_bar methods.  The default is the group name in
+#   snake_case.
+#
+# comment:
+#   A rustdoc comment that is added to the group's enums.
```

Comment:
Should this be called "rustdoc" or "doc" instead to indicate that this is the documentation for the generated enum?

Reply:
Done